src/nav-model.ts

```diff
@@ -38,6 +41,9 @@ export class NavModel {
   */
   router: Router;
 
+  /**@internal */
```

Comment:
I'm not sure if this should be internalized. I could see someone depending on it?

Reply:
Mainly added this in typing for internal usage and it does not exist in our current distribution, I think we may not need to worry about it.

Reply:
Order is part of the public api. See here for example: https://aurelia.io/docs/routing/configuration#basic-configuration